examples/dapp/package.json

```diff
@@ -17,7 +17,7 @@
       "not dead",
       "not op_mini all"
     ],
-    "development": [
+    "development": [ 
```

Comment:
Extra whitespace at the end is not needed

Reply:
Updated.
https://github.com/subspace/subspace.js/pull/7/commits/a4c452705b9649a03814358769335e8cdffca89e